exchanges/binance/binance_websocket.go

```diff
@@ -250,7 +250,7 @@ func (b *Binance) WsHandleData() {
 
 					b.Websocket.DataHandler <- exchange.TradeData{
 						CurrencyPair: currency.NewPairFromString(trade.Symbol),
-						Timestamp:    time.Unix(0, trade.TimeStamp),
+						Timestamp:    time.Unix(0, trade.TimeStamp * 1000000),
```

Comment:
Could this be changed to `time.Millisecond`?

Reply:
I'm using go.mod with the following content in order to use customized library:

```
module "<REPOSITORY_NAME>"

go 1.12.5

require (
	github.com/gorilla/mux v1.7.3
	github.com/gorilla/websocket v1.4.0
	github.com/mailru/go-clickhouse v1.1.0
	github.com/thrasher-/gocryptotrader v0.0.0-20190704200255-5b270f5ec9fd
	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb // indirect
	golang.org/x/text v0.3.2 // indirect
	golang.org/x/tools v0.0.0-20190708203411-c8855242db9c // indirect
)

// Using forked repository
// replace github.com/thrasher-/gocryptotrader => github.com/DirectX/gocryptotrader v1.0.0

// Using local folder added to .gitignore
replace github.com/thrasher-/gocryptotrader => ./vendor/gocryptotrader
```